server/tests/test_signature_adversarial.py

```
@@ -0,0 +1,219 @@
+# ---------------------------------------------------------------------------
+# Confirmed false positives (xfail until the rule is tightened)
+# ---------------------------------------------------------------------------
+
+BENIGN_KNOWN_FP = [
+    pytest.param("systemctl enable nginx.service", "persist-systemd",
+                 id="systemctl-enable-fp"),
+]
+
+
+@pytest.mark.xfail(reason="Documented over-broad rule; tighten before flipping")
+@pytest.mark.parametrize("command,expected_rule", BENIGN_KNOWN_FP)
```

Ruff (0.15.12)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)